ruby/spec/cucumber/cucumber_expressions/custom_parameter_type_spec.rb

```diff
@@ -155,7 +155,7 @@ def ==(other)
             }.to raise_error('There is already a parameter with name color')
           end
 
-          it 'is not detected for type' do
+          it 'is not detected for type', skip: 'missing expectation' do
```

Comment:
Just flagging this for me here, this should be checked across languages. It "could" be similar issue